### DISCUSSION TITLE
Does CLI support options which can be specified multiple times?

### DISCUSSION THREAD

**Discussion by skraikov** [spectreconsole/spectre.console#330] [Q&A] [answered]
Like -p option for msbuild or -D for gcc:
```bash
gcc -D CONFIG_1 -D FEATURE_2 ...
```
If yes, is there the sample for this feature?

**Comment by wayneyaoo** [comment, marked as answer]
Possible dupe of this https://github.com/spectreconsole/spectre.console/discussions/465. Haven't tried this out but per author's answer it should be supported already.
